src/js/components/navbars.js

```diff
@@ -85,48 +86,127 @@
 
   var module = angular.module('mobile-angular-ui.components.navbars', []);
 
-  /**
-   * @directive navbarAbsoluteTop
-   * @restrict C
-   * @description
-   *
-   * Setup absolute positioned top navbar.
-   *
-   * ``` html
-   *  <div class="navbar navbar-app navbar-absolute-top">
-   *    <!-- ... -->
-   *  </div>
-   * ```
-   */
+  var global;
```

Comment:
That's not my directive. It was there already.

On Sep 7, 2016 2:28 PM, "Maurizio Casimirri" <notifications@github.com>
wrote:

> In src/js/components/navbars.js
> <https://github.com/mcasimir/mobile-angular-ui/pull/378#discussion_r77907410>
> :
>
> > @@ -85,48 +86,127 @@
> >
> >    var module = angular.module('mobile-angular-ui.components.navbars', []);
> >
> > -  /**
> > -   * @directive navbarAbsoluteTop
> > -   * @restrict C
> > -   * @description
> > -   *
> > -   * Setup absolute positioned top navbar.
> > -   *
> > -   * ``` html
> > -   *  <div class="navbar navbar-app navbar-absolute-top">
> > -   *    <!-- ... -->
> > -   *  </div>
> > -   * ```
> > -   */
> > +  var global;
>
> That's quite too generic to be understood. What is it supposed to hold,
> why u need a singleton for that?
>
> I've still to read the rest but looks like a disruptive approach. We'll
> fix it anyway i like your idea of supporting common patterns.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mcasimir/mobile-angular-ui/pull/378/files/ccdae045247c4f5e44c42d1cbd5319eb3fcb252f#r77907410>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADDzgxsXu_wrzZBpkfuSbTUdmL5AuQU6ks5qnyx0gaJpZM4Jx8nS>
> .
>